packages/web-react/src/components/Tooltip/README.md

```diff
@@ -155,7 +155,7 @@ const { getReferenceProps, getFloatingProps } = useInteractions([hover, focus, d
       top: y ?? 0,
       left: x ?? 0,
     }}
-    data-placement={placement}
+    data-spirit-placement={placement}
```

Comment:
I think this is part of Floating UI. Must check!

Reply:
My bad, it is a reference to SCSS - https://github.com/lmc-eu/spirit-design-system/pull/935/files#diff-d188d47603a8e0ae11d30058f82a2f99f6d66697bab530888ffdcc52d706cb63